### ISSUE TITLE
No testem-electron.js file created

### ISSUE THREAD

**Issue by sfyfe31** [felixrieseberg/ember-electron#162]
I've got a very small Ember Electron app created with one component. The integration test fails with:

```
not ok 9 Electron (CI) - Integration | Component | file browser - it renders
    ---
        actual: >
            null
        expected: >
            undefined
        stack: >
                at assert (file:///Users/scott/Sites/V_STICK/tmp/core_object-tests_dist_electron-zIOQcmop.tmp/assets/vendor.js:16244:13)
                at Object.assert (file:///Users/scott/Sites/V_STICK/tmp/core_object-tests_dist_electron-zIOQcmop.tmp/assets/vendor.js:27912:34)
                at Class.setup (file:///Users/scott/Sites/V_STICK/tmp/core_object-tests_dist_electron-zIOQcmop.tmp/assets/vendor.js:52264:19)
        message: >
            Promise rejected before "it renders": Assertion Failed: You cannot use the same root element (DIV) multiple times in an Ember.Application
```

In trying to debug what the issue is, i noticed that `testem-electron.js` wasn't created in the root of the project, which is mentioned in the documentation:

> When running ember electron:test, testem is configured to use /testem-electron.js instead of /testem.js (to allow projects to run as traditional ember apps as well as electron apps). The blueprint will install this file, but if you want to modify it to change any testem configuration, make sure to modify the right file.

How can I create this file?

**Comment by bendemboski** [comment]
Did you install via `ember install`? If so, it should have run the blueprint and created that file. In any case, you can run `ember g ember-electron` to rerun the blueprint.

**Comment by sfyfe31** [comment]
`ember g ember-electron` doesn't generate it either. This is the output and `testem-electron.js` isn't created:

```
MacBook-Pro:EMBER_ELECTRON_APP scott$ ember g ember-electron
installing ember-electron
? Overwrite config/environment.js? Yes, overwrite
? Overwrite electron.js? Yes, overwrite
? Overwrite tests/electron.js? Yes, overwrite
  overwrite config/environment.js
  overwrite electron.js
  overwrite tests/electron.js
  identical tests/package.json
Ember Electron requires configuration. Please consult the Readme to ensure that this addon works!
https://github.com/felixrieseberg/ember-electron
```

I originally set up this app with:

```
npm install -g ember-cli
ember new EMBER_ELECTRON_APP
cd EMBER_ELECTRON_APP
ember install ember-electron
```

**Comment by sfyfe31** [comment]
I manually added the `tested-electron.js` and `test-runner.js` files, with their contents copied from this main repository. But still getting the following output when running a test:

```
MacBook-Pro:V_STICK scott$ ember electron:test
cleaning up...
Built project successfully. Stored in "/Users/scott/Sites/EMBER_APP/tmp/core_object-tests_dist_electron-Bdfji4hL.tmp".
WARNING: The `baseURL` and `rootURL` settings should not be used at the same time.
ok 1 Electron (CI) - JSHint | app.js # should pass jshint
ok 2 Electron (CI) - JSHint | components/file-browser.js # should pass jshint
ok 3 Electron (CI) - JSHint | electron.js # should pass jshint
ok 4 Electron (CI) - JSHint | helpers/destroy-app.js # should pass jshint
ok 5 Electron (CI) - JSHint | helpers/module-for-acceptance.js # should pass jshint
ok 6 Electron (CI) - JSHint | helpers/resolver.js # should pass jshint
ok 7 Electron (CI) - JSHint | helpers/safe.js # should pass jshint
ok 8 Electron (CI) - JSHint | helpers/start-app.js # should pass jshint
not ok 9 Electron (CI) - Integration | Component | file browser - it renders
    ---
        actual: >
            null
        expected: >
            undefined
        stack: >
                at assert (file:///Users/scott/Sites/EMBER_APP/tmp/core_object-tests_dist_electron-Bdfji4hL.tmp/assets/vendor.js:16244:13)
                at Object.assert (file:///Users/scott/Sites/EMBER_APP/tmp/core_object-tests_dist_electron-Bdfji4hL.tmp/assets/vendor.js:27912:34)
                at Class.setup (file:///Users/scott/Sites/EMBER_APP/tmp/core_object-tests_dist_electron-Bdfji4hL.tmp/assets/vendor.js:52264:19)
        message: >
            Promise rejected before "it renders": Assertion Failed: You cannot use the same root element (DIV) multiple times in an Ember.Application
```

`tests/integration/components/file-browser-test.js`:

```
import { moduleForComponent, test } from 'ember-qunit';
import hbs from 'htmlbars-inline-precompile';

moduleForComponent('file-browser', 'Integration | Component | file browser', {
  integration: true
});

test('it renders', function(assert) {

  // Set any properties with this.set('myProperty', 'value');
  // Handle any actions with this.on('myAction', function(val) { ... });

  this.render(hbs`{{file-browser}}`);
  assert.equal(this.$().text().trim(), '');

  // Template block usage:
  this.render(hbs`
    {{#file-browser}}
      template block text
    {{/file-browser}}
  `);

  assert.equal(this.$().text().trim(), 'template block text');
});
```

**Comment by pichfl** [comment]
Which version of ember-electron are you running?

**Comment by sfyfe31** [comment]
`ember-electron@1.12.7`

**Comment by pichfl** [comment]
This should be resolved by the 2.0 release, can you verify @sfyfe31?
